### PR TITLE
Scala → 2.12

### PR DESCRIPTION
**Pull request by regiskuckaertz**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -4,12 +4,13 @@ name := "tags-thrift-schema"
 organization := "com.gu"
 
 libraryDependencies ++= Seq(
-  "org.apache.thrift" % "libthrift" % "0.9.3",
+  "org.apache.thrift" % "libthrift" % "0.9.2",
```
Comment (itsibitzi) [inline build.sbt:7]: Why the downgrade?

File: version.sbt [hunk 2]
```diff
@@ -1 +1 @@
-version in ThisBuild := "1.1.2"
+version in ThisBuild := "1.1.3"
```
Comment (itsibitzi) [inline version.sbt]: Note to self: This should only be a version bump if the libthrift downgrade is needed.

File: build.sbt (hunk 1 above)
Comment (regiskuckaertz) [reply build.sbt:7]: because we are using 0.9.2 throughout for compatibility with scrooge 0.4.12 and also they changed the signature of `equals` somewhere

Comment (itsibitzi) [reply build.sbt:7]: Ah, ok!

Will the change to the equals signature affect users of `tags-thrift-schema` or is it purely internal?

Comment (regiskuckaertz) [reply build.sbt:7]: yeah, essentially using 0.9.3 everywhere we use scrooge leads to runtime errors

Comment (itsibitzi) [reply build.sbt:7]: Ok no worries, should probably change the build to be `2.0.0` to reflect the fact it's a non-backwards compatible API change (from the perspective of `tags-thrift-schema`)

**Review by itsibitzi — APPROVED** [review]
This looks fine to me. 

Would be nice to know what it will mean in terms of using the new thrift version. If stuff is going to break at runtime then we'll need to have some kind of test plan for at the very least `tagmanager`

**Pull request merged**